sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java

```diff
@@ -207,4 +211,36 @@ OrchestrationChatResponse maskingPseudonymization() {
 
     return client.chatCompletion(prompt, configWithMasking);
   }
+
+  /**
+   * Using grounding to provide additional context to the AI model.
+   *
+   * @link <a href="https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/grounding">SAP
+   *     AI Core: Orchestration - Grounding</a>
+   */
+  @GetMapping("/grounding")
+  @Nonnull
+  OrchestrationChatResponse grounding() {
+    final var message =
+        Message.user(
+            "{{?groundingInput}} Use the following information as additional context: {{?groundingOutput}}");
+    final var prompt =
+        new OrchestrationPrompt(
+            Map.of("groundingInput", "Give the number of letters in the word 'world'."), message);
```

Comment:
Since this is sample code, let's use a prompt where background information would actually be helpful / needed